src/strain.cmd.js

```diff
@@ -278,29 +356,53 @@ class StrainCommand {
 }`;
   }
 
+  async vclopts(name, vcl) {
+    const baseopts = await this.version(`/vcl/${name}`);
+    const postopts = await this.version('/vcl');
+    // making a get request to probe if the VCL already exists
+    return request.get(baseopts).then(() =>
+      // overwrite
+      Object.assign({
+        method: 'PUT',
+        form: {
+          name,
+          content: vcl,
+        },
+      }, baseopts)).catch(() =>
+      // create new
+      Object.assign({
```

Comment:
you should probably test whit the request failed, and only create a new one, for a 404.

Reply:
Will do.